packages/shorebird_cli/lib/src/gradlew.dart

```diff
@@ -60,12 +60,11 @@ class Gradlew {
     final javaHome = java.home;
     // Flutter apps have android files in root/android
     // Flutter modules have android files in root/.android
-    final androidRoot = [
-      Directory(p.join(projectPath, 'android')),
-      Directory(p.join(projectPath, '.android')),
-    ].firstWhereOrNull((dir) => dir.existsSync());
+    final androidRoot = Directory(p.join(projectPath, 'android'));
```

Comment:
Do we want to change this logic? Or should we just handle this explicitly in `shorebird init`?

Reply:
Discussed offline – keeping the change here